--- a/channels/client/addin.h
+++ b/channels/client/addin.h
@@ -22,7 +22,7 @@
 FREERDP_API void* channel_client_create_handler(rdpContext* ctx, LPVOID userdata,
                                                 MsgHandler handler, const char* channel_name);
 
-UINT channel_client_post_message(void* MsgsHandle, LPVOID pData, UINT32 dataLength,
-                                 UINT32 totalLength, UINT32 dataFlags);
+FREERDP_API UINT channel_client_post_message(void* MsgsHandle, LPVOID pData, UINT32 dataLength,
+                                             UINT32 totalLength, UINT32 dataFlags);
 
-UINT channel_client_quit_handler(void* MsgsHandle);
+FREERDP_API UINT channel_client_quit_handler(void* MsgsHandle);
